var/spack/repos/builtin/packages/ecp-io-sdk/package.py

```diff
@@ -16,26 +16,27 @@ class EcpIoSdk(CMakePackage):
 
     version('1.0', branch='master')
 
-    variant('hdf5', default=True, description="Enable HDF5")
     variant('adios2', default=True, description="Enable ADIOS2")
-    variant('pnetcdf', default=True, description="Enable PNetCDF")
     variant('darshan', default=True, description="Enable Darshan")
+    variant('faodel', default=False, description="Enable FAODEL")
+    variant('hdf5', default=True, description="Enable HDF5")
     variant('mercury', default=True, description="Enable Mercury")
+    variant('pnetcdf', default=True, description="Enable PNetCDF")
     variant('unifyfs', default=True, description="Enable UnifyFS")
     variant('veloc', default=True, description="Enable VeloC")
 
     # Currently no spack packages
     # variant('romio', default=False, description="Enable ROMIO")
-    # variant('faodel', default=False, description="Enable FAODEL")
 
-    depends_on('hdf5', when='+hdf5')
-    depends_on('adios2', when='+adios2')
-    depends_on('parallel-netcdf', when='+pnetcdf')
-    depends_on('veloc', when='+veloc')
-    depends_on('unifyfs', when='+unifyfs')
+    depends_on('adios2', when='+adios2+mpi+fortran')
```

Comment:
Did you mean:
```suggestion
    depends_on('adios2+mpi+fortran', when='+adios2')
```
There are no `mpi` or `fortran` variants for this package.

Reply:
I realized I was working with a local modified adios2 file.  I'll push a fix shortly.